android/build.gradle

```diff
@@ -1,50 +1,69 @@
+group 'com.example.posthog_flutter'
+version '1.0-SNAPSHOT'
+
 buildscript {
+    ext.kotlin_version = '1.7.10'
     repositories {
         google()
         mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath 'com.android.tools.build:gradle:7.3.0'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
-rootProject.allprojects {
+allprojects {
     repositories {
         google()
         mavenCentral()
     }
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
 
 android {
-    compileSdk 33
-
-    // Conditional for compatibility with AGP <4.2.
     if (project.android.hasProperty("namespace")) {
-        namespace 'com.posthog.posthog_flutter'
+        namespace 'com.example.posthog_flutter'
     }
 
-    defaultConfig {
-        minSdkVersion 19
+    compileSdkVersion 33
 
-        ndk {
-            // Flutter does not currently support building for x86 Android (See Issue 9253).
-            abiFilters("armeabi-v7a", "x86_64", "arm64-v8a")
```

Comment:
Lets keep this `abiFilters`, why is it removed?

Reply:
The issue is now closed. Look : https://github.com/flutter/flutter/issues/9253
It is the reason why abiFilters did not come with the last flutter package scaffold.

Reply:
Its closed as not supported and not fixed.
https://docs.flutter.dev/deployment/android#build-an-app-bundle

> By default, the app bundle contains your Dart code and the Flutter runtime compiled for [armeabi-v7a](https://developer.android.com/ndk/guides/abis#v7a) (ARM 32-bit), [arm64-v8a](https://developer.android.com/ndk/guides/abis#arm64-v8a) (ARM 64-bit), and [x86-64](https://developer.android.com/ndk/guides/abis#86-64) (x86 64-bit).

So the `abiFilters` don't compile or ship the `x86` which is not available anyway.